### DISCUSSION TITLE
Internal temperature

### DISCUSSION THREAD

**Discussion by Dephead2004** [BimmerGestalt/AAIdrive#721] [Ideas]
Hi Hufman,

Many thanks for all your hard work on this essential app.

I notice that the app can read the internal temperature of the car. Do you know where the sensor is and is it possible to create a display on the idrive anywhere?

I currently have a nasty small stick on thermometer. I'd love to get rid and have a respite from the idrive

**Comment by hufman** [comment]
Good morning, and thank you for the compliments!
Funny coincidence, I just promoted the beta build to the latest stable, which includes a [new car app](https://github.com/BimmerGestalt/AAIdrive/issues/653) to show several useful car data points. Check it out!
![image](https://github.com/BimmerGestalt/AAIdrive/assets/1592375/0696f231-d89f-4020-84d6-183a9f7a9a26)

I don't know where the thermometer is, for any of these data points, however.

**Reply by Organized92** [reply]
Do I understand correctly, that this is now part of the regular AAIDrive app, which is available on the Play Store?
I have version 1.4.2-3-f8e3a46 on my device, but I cannot find this menu in my car. It's a 2017 F31 with NBT Evo.
Everything else regarding AAIDrive is working perfectly.

**Reply by hufman** [reply]
Yes this was released as part of 1.4.2 (and a few beta versions before). It's the blank unlabeled icon in the ConnectedDrive Apps menu, with another labeled icon added to the Car menu. It is only added if the Notifications feature is enabled (though Notification permissions are not required).

**Reply by Organized92** [reply]
Thank you, I have enabled the Notifiactions Feature and now the car information screen is available in my car. Great!

**Comment by Dephead2004** [comment]
That's remarkable. What a coincidence. So is that the most recent version that was just downloaded? I'll check it out.
You are a star.

**Reply by Dephead2004** [reply]
Well I have it. And it works
![20230712_192053](https://github.com/BimmerGestalt/AAIdrive/assets/139343381/0e044371-8119-47a2-8695-775749decd5f)
![20230712_192112](https://github.com/BimmerGestalt/AAIdrive/assets/139343381/12b75627-a9dd-4d41-862c-5cbae894dc06)
Please note the nasty  round thermometer to be replaced by the fantastic idrive display.

What is EVAC please?

**Reply by hufman** [reply]
I'm not sure, it's an Air Conditioning [datapoint called "evaporator"](https://github.com/BimmerGestalt/AAIdrive/blob/367ccf3f1ee2730faed99d8978eefa8222e090c3/app/src/main/java/me/hufman/androidautoidrive/cds/CDSMetrics.kt#L152-L157), which is different than the heat exchanger which is shown in the HVAC field.

**Comment by Dephead2004** [comment]
Thanks I'll ask around on another forum and report back. 
Hope to get some interest in aaidrive too once I post some pictures.

**Comment by Dephead2004** [comment]
Ok so the EVAC temperature sensor records the temperature of the evaporator (the cold part of the Air Con system) turning on the aircon leads to a dramatic reduction  in the EVAC temperature.. Now to research the HVAC temp sensor

**Comment by labanauskas** [comment]
Hi, is it possible to add  option to show temperature of automatic gearbox?

Also this menu in f30 lci with i-step 2018-03 entrynav2 is shown as empty selection. I'm clicking on empty space and then detailed menu is shown.

**Reply by hufman** [reply]
The only data points that are available are listed [here](https://bimmergestalt.github.io/BMWConnectedAnalysis/cds/), and I don't see any that would show a transmission temperature.
The actual app under ConnectedDrive is an empty space, yes, because otherwise the icon and label is entirely incorrect, and there is no way to change the label. You should find a secondary icon under the Vehicle Information menu that links to this same app.
